e2e-tests/insurance/cypress/e2e/journeys/dashboard/dashboard-populated-application-non-gbp-currency.spec.js

```diff
@@ -0,0 +1,113 @@
+import dashboardPage from '../../../../../pages/insurance/dashboard';
+import partials from '../../../../../partials';
+import { PAGES } from '../../../../../content-strings';
+import { ROUTES, FIELD_VALUES } from '../../../../../constants';
+import { POLICY as FIELD_IDS } from '../../../../../constants/field-ids/insurance/policy';
+import application from '../../../../../fixtures/application';
+import { NON_STANDARD_CURRENCY_CODE } from '../../../../../fixtures/currencies';
+import formatCurrency from '../../../../../helpers/format-currency';
+
+const { table } = dashboardPage;
+
+const { DASHBOARD } = ROUTES.INSURANCE;
+
+const CONTENT_STRINGS = PAGES.INSURANCE.DASHBOARD;
+
+const { TABLE_HEADERS } = CONTENT_STRINGS;
+
+const {
+  CONTRACT_POLICY: {
+    SINGLE: { TOTAL_CONTRACT_VALUE },
+  },
+  EXPORT_VALUE: {
+    MULTIPLE: { MAXIMUM_BUYER_WILL_OWE },
+  },
+} = FIELD_IDS;
+
+context('Insurance - Dashboard - populated application - non GBP currency', () => {
```

Comment:
could this say alternative currency instead of non GBP?

Reply:
added "alternative", want to keep "non GBP" so that it's clear. We do this in a few places